src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write code here
+
+const employees = document.getElementsByTagName('li');
+
+function sortList(toBeSorted) {
+  const employeesArray = [...toBeSorted].sort((a, b) =>
+    b.dataset.salary.substring(1).replaceAll(',', '')
+  - a.dataset.salary.substring(1).replaceAll(',', ''));
+
+  const StartPoint = document.getElementsByTagName('ul');
+
+  for (let i = 0; i < employeesArray.length; i++) {
```

Comment:
Avoid using a 'for' loop. Consider using 'forEach' or 'map' for better readability and performance.